index.js

```diff
@@ -368,6 +368,14 @@ class GoogleAuth extends DbBase {
     return !!(admin.level === 0 || admin.analyticsPrivilege)
   }
 
+  async getAdmin (email) {
+    const admin = await this._getAdmin(email)
+    if (this.conf.useDB) {
```

Comment:
As a security measure using _.pick(admin, displayKeys) might be better. 
As some one might introduce a key that is not intended to be displayed and then would be shown. 
As an example, here, might be the id of the admin.